reagurk/public/index.html

```diff
@@ -4,6 +4,15 @@
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
+<!-- new meta tags-->
+      <meta property="og:type" content="website">
+      <meta property="og:url" content="<Your website link>">
+      <meta property="og:title" content="Gurkult">
+      <meta property="og:description" content="The Gurkan open source community website.">
+      <meta property="og:image" content="https://i.ibb.co/S0KVsg9/logo.png">
+      <meta property="fb:app_id" content="<Facebook App ID>" />
+      <!-- <meta name="twitter:card" content="summary_large_image"> -->
+<!--new meta tags -->
     <meta name="theme-color" content="#000000" />
```

Comment:
Can we change this to our green? `#10B981` maybe, because that's used on the main page of the site.